schema.sql

```diff
@@ -62,4 +62,12 @@ CREATE TABLE visits (
  date_of_visit DATE,
  CONSTRAINT vet_fk FOREIGN KEY(vets_id) REFERENCES vets(id),
  CONSTRAINT animals_fk FOREIGN KEY(animals_id) REFERENCES animals(id)
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+/* Create index on visits table */
+CREATE INDEX visits_animals_id_idx ON visits(id);
+CREATE INDEX visits_vets_id_idx ON visits(id);

```

Comment:
[ OPTIONAL] Please separate the screenshots with headings `BEFORE` and `AFTER` and also write the query for every screenshot. This way it would be more readable for the code reviewers to review the results.